### PR TITLE
Added git section to Coding Standards.

### PR DESCRIPTION
**Pull request by TheeDeafFrog**

### CODE REVIEW

File: Documents/Coding Standards/CodingStandards.tex [hunk 1]
```diff
@@ -34,5 +34,33 @@
   \subsection{Swift}
   For Swift we will use \href{https://swift.org/documentation/api-design-guidelines/}{Apple design guidelines}.
   
+  \section{Git Structure}
+  \subsection{Branching}
+  Rule one, always, is never work out of \texttt{master}. A new branch is created when someone wants to implement, or experiment with something that is not theirs, or in the \texttt{master}, so they will branch off of the relevant branch, to make their proposed changes.
```
Comment (Letanyan) [inline Documents/Coding Standards/CodingStandards.tex]: The always makes reading the sentence awkward, maybe consider removing it?

File: Documents/Coding Standards/CodingStandards.tex [hunk 2]
```diff
@@ -34,5 +34,33 @@
...
+  
+  \subsection{The Bulleted Rules}
+  \begin{itemize}
+   \item \textit{Never} make modifications directly to \texttt{master}.
+   \item Never directly make modifications to somebody else's code, rather branch, make the change, submit a pull request, then announce the request on Slack.
+   \item If you absolutely feel the need to work in a branch that you did not create, message the branch creator and ask for permission.
+   \item Never approve your own pull request, let the team leader, or the active benefactor to the branch you want to merge with make the merge.
+   \item Allow a sufficient amount of time and discussion to pass before merging---this applies to any merge, no matter how small, the key is in judging the time and discussion that needs to pass: the larger the merge, the larger the discussion.
```
Comment (Letanyan) [inline Documents/Coding Standards/CodingStandards.tex]: What do you mean by sufficient? This is quite vague?

File: Documents/Coding Standards/CodingStandards.tex [hunk 3]
```diff
@@ -34,5 +34,33 @@
...
   
-\end{document}
\ No newline at end of file
+  \subsection{Merging}
+  After changes have been made on the branch, the creator, who wants to merge, shall then make a pull request on GitHub, and announce the request on Slack, where it shall become the responsibility of the entire team, but more importantly that of the original branch maintainer to analyze and comment on the changes. After a sufficient amount of time and discussion has passed, either the branch maintainer (if it is a minor change), or the entire team (if it is to master, or a major change) shall decide to merge or not.
```
Comment (Letanyan) [inline Documents/Coding Standards/CodingStandards.tex]: I'm not a fan of using slack to discuss reviews. I really think we should be using GitHub reviewers and this review style to handle reviews as in this case we can make review comments inline which is much easier. Maybe slack just to notify the code owner that you have a PR.

File: Documents/Coding Standards/CodingStandards.tex (hunk 1 above)
Comment (Letanyan) [reply Documents/Coding Standards/CodingStandards.tex]: I'm not saying that the meaning of always shouldn't be there. Just that it's placement in the sentence structure is awkward to read. For example, something like "Rule one, never work out of master. Ever."

Comment (TheeDeafFrog) [reply Documents/Coding Standards/CodingStandards.tex]: I got what you meant. It's just that I disagree, I think it makes it clear that the first of working with GitHub is to never work out master, so it always applies.

Comment (TheeDeafFrog) [reply Documents/Coding Standards/CodingStandards.tex]: Ultimately, I think we need a third party to decide.

Comment (Letanyan) [reply Documents/Coding Standards/CodingStandards.tex]: Okay, then I guess we need other reviewers. I'll notify the other members that they should participate in PR reviews.

**Comment by TheeDeafFrog** [comment]
Given the Developer branch, this is redundant: closing and making new request.

**Pull request closed without merging**